src/classes/HiFiCommunicator.ts

```diff
@@ -276,40 +276,16 @@ export class HiFiCommunicator {
      * @param gain  The relative gain to apply to the other user. By default, this is `1.0`. The gain can be any value greater or equal to `0.0`.
      * For example: a gain of `2.0` will double the loudness of the user, while a gain of `0.5` will halve the user's loudness. A gain of `0.0` will effectively mute the user.
      * 
-     * @returns If this operation is successful, the Promise will resolve with {@link SetOtherUserGainForThisConnectionResponse} with `success` equal to `true`.
-     * If unsuccessful, the Promise will reject with {@link SetOtherUserGainForThisConnectionResponse} with `success` equal to `false` and `error` set to an error message describing what went wrong.
+     * @returns If this operation is successful, the Promise will resolve with {@link SetOtherUserGainsForThisConnectionResponse} with `success` equal to `true`.
+     * If unsuccessful, the Promise will reject with {@link SetOtherUserGainsForThisConnectionResponse} with `success` equal to `false` and `error` set to an error message describing what went wrong.
      */
-    async setOtherUserGainForThisConnection(visitIdHash: String, gain: Number): Promise<SetOtherUserGainForThisConnectionResponse> {
-        let setOtherUserGainForThisConnectionResponse;
-
-        if (!this._mixerSession) {
-            let errMsg = `\`this._mixerSession\` is falsey!`;
-            return Promise.reject({
-                success: false,
-                error: errMsg
-            });
-        }
-
-        try {
-            setOtherUserGainForThisConnectionResponse = await this._mixerSession.setOtherUserGainForThisConnection(visitIdHash, gain);
-        } catch (errorSettingUserGain) {
-            let errorReason : any;
-            if (errorSettingUserGain.error) {
-                errorReason = errorSettingUserGain.error;
-            } else {
-                errorReason = JSON.stringify(errorSettingUserGain);
-            }
-            let errMsg = `Error when setting other user gain for this connection. Error: \n${errorReason}`;
-            return Promise.reject({
-                success: false,
-                error: errMsg,
-                audionetSetOtherUserGainForThisConnectionResponse: errorSettingUserGain.audionetSetOtherUserGainForThisConnectionResponse
-            });
-        }
+    async setOtherUserGainsForThisConnection(otherUserGainMap: OtherUserGainMap): Promise<SetOtherUserGainsForThisConnectionResponse> {
```

Comment:
@sabrina-shanman Sorry, one final thing -- because people already have `setOtherUserGainForThisConnection` in production, can you please add a wrapper function of `async setOtherUserGainForThisConnection(visitIdHash: String, gain: Number): Promise<SetOtherUserGainForThisConnectionResponse>` to this for backwards compatibility and the "single user" use case?

In other words, there should continue to be a `setOtherUserGainForThisConnection` method, but internally it should just create the single-user OtherUserGainMap and hand it into this one.

Reply:
Oh, right! I'll add that back.

Reply:
Similarly...if it's possible, it would be great to add an alias of `SetOtherUserGainForThisConnectionResponse` for the newly renamed `SetOtherUserGainsForThisConnectionResponse`. If that's not possible, I think my preference would be to leave it singular (for backwards compatibility) despite the naming difference with the primary (plural) method.